<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="qt_tblSourceCatalog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>371</width>
      <height>621</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="qt_txtedTelescopeParams">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="qt_lstObsSchedule">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="qt_calStartDate">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>264</width>
      <height>165</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="qt_lblObservationSchedule">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Observation Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="qt_btnGenerateSchedule">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>790</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generate FS Schedule!</string>
    </property>
   </widget>
   <widget class="QLabel" name="qt_lblStartDate">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start date:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="qt_txtReservedForPlot">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans [monotype]'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;I envision a matplotlib widget sitting in this space.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>800</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="qt_lblOutputFilename">
     <property name="text">
      <string>Output filename</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qt_lineOutputFilename"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="qt_lblTimeZone">
     <property name="text">
      <string>Time zone:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qt_cmbTimeZone"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="qt_lblStartTime">
     <property name="text">
      <string>Start time:</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="qt_timeStartTime">
     <property name="displayFormat">
      <string>HH:MM</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="qt_lblTelescopeParams">
     <property name="text">
      <string>Telescope Parameters</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qt_btnOpenTelescopeParams">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>126</y>
      <width>371</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="qt_lblSourceCatalog">
     <property name="text">
      <string>Source catalog:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="qt_btnOpenSourceCatalog">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>371</width>
      <height>26</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QComboBox" name="qt_cmbObsType"/>
    <widget class="QPushButton" name="seesc">
     <property name="text">
      <string>Insert obs on target</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="qt_btnSetToNow">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Set to now</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
